web-admin/src/components/home/DashboardList.svelte

```diff
@@ -16,6 +16,13 @@
   }
 
   async function updateDashboardsForProject() {
```

Comment:
It'd be better if this function took `projectData` as a parameter, rather than closing over the global value.